lib/monit_interface.py

```diff
@@ -127,3 +134,46 @@ def restart(watch):
 
   logging.info("Restarting watch {0}".format(watch))
   return run_with_retry([MONIT, 'restart', '-g', watch])
+
+def create_monit_config_file(watch, start_cmd, stop_cmd, ports, match_cmd,
```

Comment:
Is there a reason you didn't use `create_config_file` in `lib/monit_app_configuration.py`?